composer.json

```diff
@@ -4,13 +4,13 @@
     "type": "project",
     "description": "Primary dependencies for XOOPS/XoopsCore25",
     "require": {
-        "php": ">=5.6.0",
-        "xoops/xmf": "^1.2.29",
+        "php": ">=7.4.0",
+        "xoops/xmf": "^1.2.30",
         "ircmaxell/password-compat": "^1.0.4",
```

Comment:
Since we are requiring PHP >=7.4.0 we can remove this dependency as the functionality is built in.

Reply:
OK, I've deleted it